.github/workflows/build_and_deploy.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         run: |
           make ci review-aks get-cluster-credentials
           kubectl exec -n srtl-development deployment/claim-additional-payments-for-teaching-review-${{ github.event.pull_request.number }}-worker -- sh -c "DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/prepare-database"
+          kubectl exec -n srtl-development deployment/claim-additional-payments-for-teaching-review-${{ github.event.pull_request.number }}-worker -- sh -c "bin/rails runner 'SchoolDataImporterJob.perform_later'"
```

Comment:
We have a rake task: `schools_data:import`. It would probably be good to create a separate bin script which can be used to perform seeding on review apps specifically, rather than specifying it in the workflow file.